### ISSUE TITLE
Unable to connect to server when bindings 1.1 and 1.0 are present

### ISSUE THREAD

**Issue by nmgrange** [PaloAltoNetworks/minemeld-core#221]
Minemeld is unable to connect to a taxii server that is distributing both message binding 1.0 and 1.1. The error returned is "<Miner> - collection <collection> does not support TAXII 1.1 message binding (DATA_FEED)"

**Comment by jtschichold** [comment]
Hi @nmgrange,
is the TAXII Server providing both STIX 1.0 and 1.1 or just STIX 1.1 ?
I am starting to work on a major reimplementation of the TAXII Miner and this could be an additional requirements. See  #205

**Comment by nmgrange** [comment]
The server is providing both 1.0 and 1.1 and the inbox will accept any content. Here is a screenshot of the inbox. When I changed it to only accept 1.1 the binding was still the same and I cannot change that.
![image](https://cloud.githubusercontent.com/assets/29075253/26689115/7ba2b562-46a9-11e7-90cc-268d4e822959.png)

And I added some logging to see what was being pulled and here is the binding that it sees
![image](https://cloud.githubusercontent.com/assets/29075253/26689241/dee194c2-46a9-11e7-92c2-7537312b4ed3.png)
